src/components/claims/claim-details/components/ClaimInformation.tsx

```diff
@@ -245,6 +246,13 @@ export const ClaimInformation: React.FC<Props> = ({ claimId, memberId }) => {
               ⚠️ No agreement covers the claim on the date of loss
             </Paragraph>
           )}
+          {contracts?.length === 0 && trials?.length > 0 ? (
+            <DangerCard>
+              This member has a trial which may cover this claim, but no
+              contract. This case can not be handled in Hope yet, please contact
+              the MX tech team.
+            </DangerCard>
+          ) : null}
```

Comment:
Don't need the question marks since you default to an empty list above. Also if you fallback to `null` you can use `&&` instead of the ternary operator, making the a bit more tidier
```suggestion
          { (contracts.length === 0 && trials.length > 0) && (
            <DangerCard>
              This member has a trial which may cover this claim, but no
              contract. This case can not be handled in Hope yet, please contact
              the MX tech team.
            </DangerCard>
          )}
```